uk.co.compucorp.civicrm.hrleaveandabsences/scss/overrides/_variables.scss

```diff
@@ -0,0 +1,7 @@
+@import "SCSSROOT/org.civicrm.shoreditch/scss/bootstrap/overrides/variables";
+@import "SCSSROOT/org.civicrm.shoreditch/base/scss/vendor/bootstrap/variables";
+
+$chr-table-condensed-cell-padding: 8px 5px;
```

Comment:
I think these variables are used under the tables partials only, then we should move this to that partial only.